content/Hardware Support/Generic/Set-a-UNO-Rev1-and-Rev3-or-Mega-board-to-DFU-Device-Firmware-Update-mode.md

```diff
@@ -17,7 +17,7 @@ The board will reset to DFU mode.
 
 <h2 id="rev1">Short the HWB line (Rev1 models)</h2>
 
-If you're using an Rev1 model, you also need to assert and hold the **HWB line**:
+If you're using a Rev1 model, you also need to assert and hold the **HWB line**:
```

Comment:
Although we generally group Rev1 and Rev2 in this article, I believe this is correct. A 1k pull-down resistor was added to the HWB line in Rev2. Just in case you were wondering.